rest-api/test/client_test/participant_test.py

```diff
@@ -21,24 +20,30 @@ def test_create_and_modify_participant(self):
         "reference": "Organization/PITT",
       }
     }
-
     # Create a new participant.
     participant = {
-        'providerLink': [provider_link]
+        'providerLink': [provider_link_2]
     }
 
     response = self.client.request_json('Participant', 'POST', participant)
-    self.assertJsonEquals(response['providerLink'], [provider_link])
+    self.assertJsonEquals(response['providerLink'], [provider_link_2])
     biobank_id = response['biobankId']
 
     participant_id = response['participantId']
 
     # Fetch that participant.
     response = self.client.request_json('Participant/{}'.format(participant_id))
-    last_etag = self.client.last_etag
+    # Test that hpo is == current provider link
+    self.assertEqual(response['hpoId'], 'PITT')
```

Comment:
May want to check response['awardee'] here and below as well.

Reply:
done.